src/index.ts

```diff
@@ -41,7 +44,7 @@ export default async function GleeAppInitializer () {
     showAppDir: GLEE_PROJECT_DIR !== process.cwd(),
     showFunctionsDir: GLEE_FUNCTIONS_DIR !== resolve(GLEE_DIR, 'functions'),
   })
-
+  const gleeDocs = await readGleeConfig(GLEE_CONFIG_FILE_PATH)
```

Comment:
I don't think you need the `readGleeConfig` function. You already have the config loaded a few lines before.